validator/src/admin_rpc_service.rs

```diff
@@ -64,6 +66,14 @@ impl AdminRpcRequestMetadata {
             ))
         }
     }
+
+    pub fn update_keypair(&self, key: &Keypair) {
+        for n in self.notifies.iter() {
+            if let Err(err) = n.update_key(key) {
+                warn!("Error updating network layer keypair: {}", err);
```

Comment:
This should be `error!`.
also, https://rust-lang.github.io/rust-clippy/master/#/uninlined_format_args